core/modules/layout/includes/block.class.inc

```diff
@@ -5,6 +5,13 @@
  */
 class Block extends LayoutHandler {
 
+  /**
+   * If this block is disabled.
+   *
+   * @var bool
+   */
+  var $status = 1;
```

Comment:
Status should be TRUE since it's boolean. Like here: https://github.com/backdrop/backdrop/blob/57bc2e7b22a08a412a5a44e07b7a2fc21fec28e2/core/modules/layout/includes/layout.class.inc#L89. Plus it probably doesn't need to be set here since it's set in the construct below.

Reply:
This is already marked resolved, but using integer is consistent with Node, File, and User objects. Not sure why we ended up with the reverse `$disabled` property on the Layout itself, perhaps to match Views?